### PR TITLE
[SYCL] Test for sycl::half operator++ (customer report followup)

### PR DESCRIPTION
**Pull request by KseniyaTikhomirova**
Signed-off-by: Tikhomirova, Kseniya <kseniya.tikhomirova@intel.com>

### CODE REVIEW

**Comment by KseniyaTikhomirova** [comment]
pre-ci-linux:   select_device failed on opencl -> not related

**Comment by KseniyaTikhomirova** [comment]
test failures are unrelated. @cperkinsintel, @intel/llvm-reviewers-runtime kindly ping

File: SYCL/Regression/half_operators.cpp [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  for (size_t i = 0; i < size; ++i) {
+    result &= lhs_bytes[i] == rhs_bytes[i];
+  }
+  return result;
+}
+
+template <typename T> bool test() {
+  sycl::queue queue{};
+
+  constexpr int NumElems{32};
```
Comment (dm-vodopyanov) [inline SYCL/Regression/half_operators.cpp:32]: nit: num_elems

**Review by dm-vodopyanov — APPROVED** [review]

**Comment by againull** [comment]
Failures:
SYCL :: Config/select_device.cpp
SYCL :: SubGroupMask/Basic.cpp

are known and tracked.

**Pull request merged**